README.md

```diff
@@ -22,26 +22,33 @@ ___
 ___
 
 ## Introduction
+
 Anomalib is a deep learning library that aims to collect state-of-the-art anomaly detection algorithms for benchmarking on both public and private datasets. Anomalib provides several ready-to-use implementations of anomaly detection algorithms described in the recent literature, as well as a set of tools that facilitate the development and implementation of custom models. The library has a strong focus on image-based anomaly detection, where the goal of the algorithm is to identify anomalous images, or anomalous pixel regions within images in a dataset. Anomalib is constantly updated with new algorithms and training/inference extensions, so keep checking!
 
-##### Key features:
+**Key features:**
+
 - The largest public collection of ready-to-use deep learning anomaly detection algorithms and benchmark datasets.
 - [**PyTorch Lightning**](https://www.pytorchlightning.ai/) based model implementations to reduce boilerplate code and limit the implementation efforts to the bare essentials.
 - All models can be exported to [**OpenVINO**](https://www.intel.com/content/www/us/en/developer/tools/openvino-toolkit/overview.html) Intermediate Representation (IR) for accelerated inference on intel hardware.
 - A set of [inference tools](#inference) for quick and easy deployment of the standard or custom anomaly detection models.
 
 ___
+
 ## Getting Started
+
 The repo is thoroughly tested based on the following configuration.
-*  Ubuntu 20.04
-*  NVIDIA GeForce RTX 3090
+
+- Ubuntu 20.04
+
+- NVIDIA GeForce RTX 3090
```

Comment:
Should we still keep this? We could either remove this bit, or add that we also support other platforms.

Reply:
Not sure. Maybe we can keep a growing list in docs rather than over here and we can link that page here.